packages/cyberstorm/src/components/Avatar/AvatarButton.tsx

```diff
@@ -22,31 +26,67 @@ export const AvatarButton = React.forwardRef<
     colorScheme,
     size = "medium",
     username,
+    icon,
     ...forwardedProps
   } = props;
 
   const fallbackRef = useRef(null);
   const ref = forwardedRef || fallbackRef;
 
-  return (
-    <button
-      className={classnames(styles.root, getSize(size))}
-      {...forwardedProps}
-      ref={ref}
-    >
-      {src ? (
+  if (src) {
```

Comment:
Well most if not all comments I left to the Button-rework PR in this PR stack apply to this component as well.

Reply:
@Oksamies what is this combination of emoji trying to say?
![image](https://github.com/thunderstore-io/thunderstore-ui/assets/3500016/4922e4ff-a1a2-48e3-98ee-c7be72124c9d)